Source/WebCore/loader/SubresourceLoader.cpp

```diff
@@ -181,14 +181,15 @@ void SubresourceLoader::init(ResourceRequest&& request, CompletionHandler<void(b
     ResourceLoader::init(WTFMove(request), [this, protectedThis = Ref { *this }, completionHandler = WTFMove(completionHandler)] (bool initialized) mutable {
         if (!initialized)
             return completionHandler(false);
-        if (!m_documentLoader) {
+        auto documentLoader = protectedDocumentLoader();
```

Comment:
The preferred pattern is:
```
RefPtr documentLoader = m_documentLoader;
```
or
```
RefPtr documentLoader = this->documentLoader();
```

We prefer not to use `auto` to hide smart pointers anymore as we want to make it clear we're using smart pointers and the code is safe.

Reply:
Do you think we need `protected` prefix here? https://webkit.org/code-style-guidelines/#names-protectors

Reply:
No, protectors are no longer a thing with our new smart pointer idioms. This is just a plain local variable. And protectedThis protectors are no longer useful since we now expect the call sites to protect the object they're calling a member function on.